tests/unit/amo/components/TestLandingAddonsCard.js

```diff
@@ -106,6 +107,77 @@ describe(__filename, () => {
     expect(root.find(AddonsCard)).toHaveProp('footerLink', null);
   });
 
+  it('hides the footer link when there are less add-ons than LANDING_PAGE_THEME_COUNT', () => {
+    const addons = [
+      createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        slug: 'custom-addon',
+      }),
+    ];
+    const root = render({ addons, isTheme: true });
+    expect(root.find(AddonsCard)).toHaveProp('footerLink', null);
+  });
+
+  it('shows the footer link when there are more or as many add-ons than LANDING_PAGE_THEME_COUNT', () => {
+    const addons = [
+      createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        slug: 'custom-addon',
+      }),
+      createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        slug: 'custom-addon-1',
+      }),
+      createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        slug: 'custom-addon-2',
+      }),
+    ];
+    const root = render({ addons, isTheme: true });
+    expect(root.find(AddonsCard)).not.toHaveProp('footerLink', null);
+
+    expect(root.find(AddonsCard).prop('footerLink')).toMatchObject({
+      key: null,
+      ref: null,
+      props: {
+        to: { pathname: '/some-path/', query: {} },
+        children: 'some text',
+      },
+      _owner: null,
+      _store: {},
+    });
+  });
+
+  it('shows the footer link when there are as many or more add-ons than placeholderCount', () => {
```

Comment:
The comments I made about `it.(shows the footer link when there are more or as many add-ons than LANDING_PAGE_THEME_COUNT)` apply to this one as well.